import/README.md

```diff
@@ -97,3 +97,27 @@ You can copy and paste the results of `inspectdb` into your models by hand, or,
 When your models are created, you will also have to go through the usual Django steps of adding them to `INSTALLED_APPS` in the site settings and [making them editable in the admin](https://docs.djangoproject.com/en/3.1/intro/tutorial02/#make-the-poll-app-modifiable-in-the-admin).
 
 
+## Importing cleaned data
+
+First, you'll need a JSON fixture. If the data has already been in a database, Django's `dumpdata` will create one for you.
```

Comment:
Could we line wrap our markdown files? I find that it makes tidier diffs in Github when a change affects only a single line of the text rather than appearing to affect the entire paragraph. I'm happy to agree to disagree on this too.

Reply:
nah, I like them wrapped, too. I just don't always think about it when I'm getting the thoughts outta my head.